function-support.html

```diff
@@ -579,7 +591,7 @@ <h2 id="environments">Environments</h2>
     <div class="indented">
 		<p>KaTeX also supports <code>{darray}</code> and <code>{dcases}</code>.</p>
 	
-        <p>Acceptable line separators include: <code>\\</code>, <code>\cr</code>, and <code>\\[<em>distance</em>]</code>. <em>Distance</em> can be written with any of the <a href="#units">KaTeX units</a>.</p>
+        <p>Acceptable line separators include: <code>\\</code>, <code>\cr</code>, and <code>\cr[<em>distance</em>]</code>. <em>Distance</em> can be written with any of the <a href="#units">KaTeX units</a>.</p>

```

Comment:
Mentioning `\cr[distance]` seems good, but `\\[distance]` is valid too, right?

Reply:
Issue #1075 occurs when the auto-render extension encounters a `\\[0.3em]` row separator inside a pair of `\[...\]` math delimiters. Until we fix that bug, I thought I would nudge people toward `\cr[0.3em]`.

Reply:
Ah, good point.  Sounds good, as long as you'll remember to add it back once #1075 gets solved.